source/quick-start.txt

```diff
@@ -14,6 +14,49 @@ Quick Start
 
 .. include:: /includes/quick-start/overview.rst
 
+Create a MongoDB Cluster
```

Comment:
Moved this section first so it goes from Create Cluster > Create Project > Code Project, rather than Create Project > Create Cluster > Code Project.